tools/cldr-apps/src/main/java/org/unicode/cldr/web/SearchManager.java

```diff
@@ -211,6 +211,12 @@ private void addCodes(final String locale) {
             logger.finest(() -> "AddCodes made resolved " + locale);
 
             for (final String x : resolvedFile.fullIterable()) {
+                if (x.startsWith("//ldml/annotations/annotation")
+                        && x.contains("[@cp=\"" + request.value + "\"]")
+                        && x.contains("[@type=\"tts\"]")) {
+                    response.addResult(new SearchResult(x, "tts: " + request.value, locale));
```

Comment:
Note that when you search for emoji that the user enters, you have to remove all the emoji variation selectors. You might already do this somewhere else.